### DISCUSSION TITLE
Community Call with Dom on June 12th - community Questions

### DISCUSSION THREAD

**Discussion by Phyloiota** [iota-community/Community-Governance#31] [Q&A]
Dom will explain on June 12th (time to be released) during a community call in the Discord General Voice channel, what the planned implementation for the first vote will look like.
The IF seems to have developed a voting Function for the Firefly wallet, which will be used to do the first big decision in this process. We would be happy to hear about the technique how the vote should happen.

We will also try to give a summary about the current state of the community discussion here in the Github and on Discord.

**To remind everyone, the planned first vote will be the decision of the community between 2 options:**

**A)     Use the tokens that are locked in the Treasury Wallet for a community governed Fund**

**B)     Destroy / Burn those tokens and remove them from the Ledger**

For detailed info please read the IOTA blogpost regarding those tokens
https://blog.iota.org/unclaimed-tokens-community-treasury-and-iota-as-faq/

**Please post Questions here below that you might find useful to ask Dom during the call!**
**Support Questions with the Thumbs Up Emoji**

**Comment by ebk1960** [comment]
I think it would be helpful if the IF explained how they see the voting mechanism actually working BEFORE the call on the 12th. That would allow the community to discuss it before the 12th and to get broad community feedback on their technical approach, prior to the call.

**Comment by HieronymusdB** [comment]
For me to be able to do a vote, I would like to have a more defined "Goal"or "Purpose" for the Fund.

now it is stated that the Fund will "fund and accelerate the growth of the IOTA ecosystem."
What type of projects /grant proposals will be eligible to get funding out of the treasury and how does this relate to the Ecosystem fund?

**Reply by basvansambeek** [reply]
This is still something we need to define and put up for a vote. Or perhaps find a model that ticks boxes for everyone.

There have been many ideas, but they’ve not been formalized yet. I’m trying to find time to do a first draft, but time is scarce. 

And once we have the basic idea, we need to set up a system of governance to determine how to vote on what. To me we’re still quite a few steps away from even the initial vote, at least if we want this initiative to have proper backing from “the community” (see another thread on this forum where we’re figuring out who is in our community exactly.)

Eventually it will come down to the people who are there that will set all this up, but we’re not there yet in my opinion.

**Reply by Deep-Sea-888** [reply]
Just my thoughts, but I think any project should be allowed to be proposed as long as it supports the growth of the IOTA ecosystem in a positive manner. That is anything from development in the protocol, developing Dapps, and also non-development projects. Such as, marketing campaigns, paying to get listed on an exchange which the IF can not pay due to their non-profit status. Or possibly funding social programs such as teaching programmers in 3rd world countries how to develop on the Tangle and provide IoT devices. I think the main standard the DAO should follow that any project does not do direct harm to individuals, animals, or the environment and supports the growth of the IOTA Protocol or Ecosystem.

For the question of how does this relates to the Ecosystem Fund, I would say this has nothing to do with the IFs Ecosystem fund. People could simultaneously propose their project to both the EDF and the DAO at the same time. Of course, somewhere in the DAO would be written, "is your project currently being funded by any other program"? To deter someone that has a great proposal and trying to get fund from multiple organizations.

**Reply by ebk1960** [reply]
Shouldn't we fund the development of a DEX (Decentralised Exchange) for P2P IOTA trading rather than use scarce DAO funds to buy exchange listings - that still comes with their existing fees structure? I can't think of any better project to show off IOTA's feeless structure than enabling the IOTA community to trade with each other without fees - after all, isn't that what IOTA is all about?

**Reply by Phyloiota** [reply]
Sure so, also - paying for exchange listings is an often misunderstood thing. None of the interesting exchanges for IOTA have such a thing. Mark Schmidt also confirmed this, so neither Coinbase nor Kraken can be bought to list IOTA. But of course a DEX developed by the community will be a great example.

**Comment by Phyloiota** [comment]
For everyone.
The call will happen tomorrow June 12th at european noon 12 am CEST and is open end
It will happen in the IOTA Discord #general-voice channel and it will also be a live stream in Youtube available here:
https://www.youtube.com/watch?v=SOY3reRHus4

Agenda:
1.) Welcome and introduction - the purpose of the Meeting (Phylo / Dom)
2.)Short recap about the fund situation
3.) what are the 2 options to vote for (governance treasury and burn) and what would they mean exactly (Phylo)
4.) Summary of what has been discussed so far in the several groups / forums / discord (Bas)
5.) Q&A popular community questions from socials (Dom / Phylo)
6) short summary of other systems (Phylo / maybe IF Team) 
7.) Ideas for possible voting solutions in IOTA / Firefly (Phylo/ IF team / everyone)
8.) How to continue from here, further regular meetings / workgroups etc
9.) open end community discussion

Discord invite for everyone
https://discord.iota.org/

**Comment by superzman1** [comment]
Hypothetically, if the tokens were burned, is there realistically going to be another opportunity to have a community governed fund? If so how would this be funded? I'm voting under the impression the IF would never create another tangle for development/community project purposes like Kusama is for Polkadot, please clarify

**Reply by Phyloiota** [reply]
I dont see any realistic chance for another Fund. Only if someone with too much money decides to donate. 
Testnets for development will always be there, thats not an issue, but such an amount of funds under the control of the community to be used for all kind of amazing developments will never happen again i think... No one has the financial backups to pull 85 million UsDollar and donate it for the good of IOTA.
Burn it and it will never be possible again to do anything so big and so good...

**Comment by MMP2019** [comment]
Some discussions were raised during the call in relations to 2/3 vs 50+ for community vote. What happens if the vote falls between the two figures? Does it mean that we won't have any resolution and that we need to revote? Thanks!

**Reply by basvansambeek** [reply]
I’d like to frame it differently, with adaptive thresholds. 

I’d say certain proposals or funding requests require higher scrutiny/consensus. We then leave it up to the proposer to determine how much funding they need/how confident they are they’ll reach consensus at certain thresholds. And if a vote does not reach the threshold at a certain time, it gets discarded. With the stake lost to prevent Sybil attacks/voter fatigue.

For governance decisions we could have something like an adaptive threshold, that starts high in required quorum, but will decrease in time to break a deadlock from low voter commitment. Then it becomes a reliable process that cannot easily be overtaken, but we also don’t lock ourselves into a certain set of rules.

**Comment by Phyloiota** [comment]
So we had a successful first call!
Here is the link to the recording on YouTube:
https://youtu.be/SOY3reRHus4

and James wrote an amazing summary here:
https://medium.com/@jamesjdsutton/iota-to-burn-or-to-build-81de0b572eb5

**Comment by AXBBXA** [comment]
Dear all,

I have been loosely following the discussion and have a couple of questions which almost certainly have already been asked and answered plenty of times. All of them concern the first vote:

1. The first post states that "_To remind everyone, the planned first vote will be the decision of the community between 2 options_: ["Build" or "Burn"]". I am not sure I have understood why that limitation applies and where it comes from. Who defined these options and why was he or she entitled to do so?
2. If the first vote is about a decision between "Build" and "Burn", should this vote not also include an option to "Do Nothing" (keep the status quo)?
3. As an alternative first vote, why not ask the question whether the community would like to assume control of the tokens (to be answered with a simple "Yes" or "No"). The second vote could then be about what to do with them ("Build", "Burn", or "Do Nothing").
4. What are the views about the quorum and the majority required for a valid decision? In order to legitimize the decision, the "community" should make sure that the first decision, whatever it will be, is not made by 13 people with only 25 voters participating. In other words, it should be made sure that there is no silent majority that was not properly invited to the vote and therefore did not partipate in it. Moreover, it should also be made sure that there is no silent majority that does not participate in the vote because it does not want to support either of the two offered options ("Build" or "Burn") and there is no alternative "Do Nothing" option.
5. My understanding is that a "Burn" decision, once executed, would be irreversible, while both the other option ("Build") and a potential "Do Nothing" option do not appear to exclude that the community decides to "Burn" the tokens later on. Am I correct? If so, in what way would a "Burn" decision as a first vote be a wise choice?

All answers and comments will be greatly appreciated - many thanks in advance!

**Reply by ebk1960** [reply]
1. There was/is a very long, tortuous Discord conversation about this very question. To get to your point, the discussion led us (those engaged around the initiation of a Treasury DAO) that believed the first vote should be simplified to just be about Build or Burn. I'll come to the 'Do Nothing'  idea in a moment. This is because, in all, I would estimate active persons engaged in the Treasury DAO Discord conversation to be around 20-25 people in total. However, we realised that those not actively engaged in the discussion (the vast majority) should therefore be presented with a straightforward decision, as otherwise they may be less likely to vote if they were required to trawl through all the various discussion on voting permutations to get their feedback - and  it is very unlikely that many would actually want do that.

**Reply by ebk1960** [reply]
2. There is always the option after the first vote (if its build) to then decide later to do nothing - if that is what the community wants. However, IMHO I cannot see the point of that. Why would the community want to do nothing if had already voted to build?

**Reply by ebk1960** [reply]
3. I think that is a valid point. Why don't you argue for that?

**Reply by ebk1960** [reply]
4. There are technical limitations to 'on tangle' voting via multiple Firefly wallets. There are also limitations to identify individual voters - a hot topic of debate on the DAO Treasury Discord channel. As it is currently deemed technically impossible to have a one person, one vote mechanism, the DAO genesis vote will therefore use a one token, one vote mechanic. While this is far from perfect, (whales get a big say in the vote) it's the best that can be done while opening the vote to anyone that holds IOTA tokens in a Firefly wallet (obviously this will be undertaken post Ledger integration). The process for voting via the Firefly wallet is outlined quite well in this Google Doc - https://docs.google.com/document/d/1Dox1-cwkv6e4sxhypA8N1jOUmMCEv9SqnUdwYMUIO5I/edit

**Reply by ebk1960** [reply]
5. Yes, the 'Burn' decision would be irreversible.

**Reply by ebk1960** [reply]
The actual technical approach to voting is outlined by Thoralf (great credit to him) here:  https://github.com/Thoralf-M/iota-voting

**Reply by AXBBXA** [reply]
Thank you very much for your comments, EBK!

> 
> 
>     1. There was/is a very long, tortuous Discord conversation about this very question. To get to your point, the discussion led us (those engaged around the initiation of a Treasury DAO) that believed the first vote should be simplified to just be about Build or Burn. I'll come to the 'Do Nothing'  idea in a moment. This is because, in all, I would estimate active persons engaged in the Treasury DAO Discord conversation to be around 20-25 people in total. However, we realised that those not actively engaged in the discussion (the vast majority) should therefore be presented with a straightforward decision, as otherwise they may be less likely to vote if they were required to trawl through all the various discussion on voting permutations to get their feedback - and  it is very unlikely that many would actually want do that.

I agree that the first vote should be as simple as possible. This is why I suggested that it may be preferable to put the question whether the "community" wishes to take control of the tokens, which can be answered with a simple "Yes" or "No".

> 
> 
>     2. There is always the option after the first vote (if its build) to then decide later to do nothing - if that is what the community wants. However, IMHO I cannot see the point of that. Why would the community want to do nothing if had already voted to build?

To be clear: The "Do Nothing" option should be offered as an alternative to the "Build" and "Burn" options, in my view, for the following reason:

Assume, for the sake of argument, that the majority prefers that the unclaimed tokens may still be claimed, for a certain time period or otherwise, by the original holders. What option will the majority choose if the choice is  limited to "Build" or "Burn"? Or is the majority not supposed to particpate in this scenario?

As far as your question is concerned ("_Why would the community want to do nothing if had already voted to build?_"). I guess it is not only that people tend to change their views. It is also that the "community" itself will change as quickly as the token holders. 

> 
> 
>     3. I think that is a valid point. Why don't you argue for that?

I am doing it right here and now!  ;-)

> 
> 
>     4. There are technical limitations to 'on tangle' voting via multiple Firefly wallets. There are also limitations to identify individual voters - a hot topic of debate on the DAO Treasury Discord channel. As it is currently deemed technically impossible to have a one person, one vote mechanism, the DAO genesis vote will therefore use a one token, one vote mechanic. While this is far from perfect, (whales get a big say in the vote) it's the best that can be done while opening the vote to anyone that holds IOTA tokens in a Firefly wallet (obviously this will be undertaken post Ledger integration). The process for voting via the Firefly wallet is outlined quite well in this Google Doc - https://docs.google.com/document/d/1Dox1-cwkv6e4sxhypA8N1jOUmMCEv9SqnUdwYMUIO5I/edit

Thank you for the information. I am aware of the "one vote for one warm body" discussion. However, I am concerned about a different issue: To ensure that the voters participating in the vote represent a sizeable share of the community.

> The actual technical approach to voting is outlined by Thoralf (great credit to him) here: https://github.com/Thoralf-M/iota-voting

Thank you again for your views and all the information. I will gladly look into it!

**Reply by ebk1960** [reply]
That is a shared concern that everyone working to bring the DAO into reality is aware of. I would suggest you help get the word out far and wide.

**Reply by Phyloiota** [reply]
> Thank you very much for your comments, EBK!
> 
> > ```
> > 1. There was/is a very long, tortuous Discord conversation about this very question. To get to your point, the discussion led us (those engaged around the initiation of a Treasury DAO) that believed the first vote should be simplified to just be about Build or Burn. I'll come to the 'Do Nothing'  idea in a moment. This is because, in all, I would estimate active persons engaged in the Treasury DAO Discord conversation to be around 20-25 people in total. However, we realised that those not actively engaged in the discussion (the vast majority) should therefore be presented with a straightforward decision, as otherwise they may be less likely to vote if they were required to trawl through all the various discussion on voting permutations to get their feedback - and  it is very unlikely that many would actually want do that.
> > ```
> 
> I agree that the first vote should be as simple as possible. This is why I suggested that it may be preferable to put the question whether the "community" wishes to take control of the tokens, which can be answered with a simple "Yes" or "No".
> 
> > ```
> > 2. There is always the option after the first vote (if its build) to then decide later to do nothing - if that is what the community wants. However, IMHO I cannot see the point of that. Why would the community want to do nothing if had already voted to build?
> > ```
> 
> To be clear: The "Do Nothing" option should be offered as an alternative to the "Build" and "Burn" options, in my view, for the following reason:
> 
> Assume, for the sake of argument, that the majority prefers that the unclaimed tokens may still be claimed, for a certain time period or otherwise, by the original holders. What option will the majority choose if the choice is limited to "Build" or "Burn"? Or is the majority not supposed to particpate in this scenario?
> 
> As far as your question is concerned ("_Why would the community want to do nothing if had already voted to build?_"). I guess it is not only that people tend to change their views. It is also that the "community" itself will change as quickly as the token holders.
> 
> > ```
> > 3. I think that is a valid point. Why don't you argue for that?
> > ```
> 
> I am doing it right here and now! ;-)
> 
> > ```
> > 4. There are technical limitations to 'on tangle' voting via multiple Firefly wallets. There are also limitations to identify individual voters - a hot topic of debate on the DAO Treasury Discord channel. As it is currently deemed technically impossible to have a one person, one vote mechanism, the DAO genesis vote will therefore use a one token, one vote mechanic. While this is far from perfect, (whales get a big say in the vote) it's the best that can be done while opening the vote to anyone that holds IOTA tokens in a Firefly wallet (obviously this will be undertaken post Ledger integration). The process for voting via the Firefly wallet is outlined quite well in this Google Doc - https://docs.google.com/document/d/1Dox1-cwkv6e4sxhypA8N1jOUmMCEv9SqnUdwYMUIO5I/edit
> > ```
> 
> Thank you for the information. I am aware of the "one vote for one warm body" discussion. However, I am concerned about a different issue: To ensure that the voters participating in the vote represent a sizeable share of the community.
> 
> > The actual technical approach to voting is outlined by Thoralf (great credit to him) here: https://github.com/Thoralf-M/iota-voting
> 
> Thank you again for your views and all the information. I will gladly look into it!

Hey, thanks for bringing these points up.

I would like to add something regarding the option `do nothing` 
Actually from a technical viewpoint the option "Burn - is exactly this - do nothing and leave the status quo. In the current running Mainnet the tokens are already burnt - this means- a block is implemented in the current node software that makes it impossible for those Tokens to be created (migrated) into the current mainnet. So if we vote for Burn / do nothing, this code will become a permanent irreversible part of the node software and those tokens will never be created in IOTA again.
"Build" would require an active action by all Node operators, to implement a new Snapshot into the Node Software that makes the creation of those Tokens possible. 
The two options have been brought up by the IOTA Foundation and the Node operators. And I believe it is a good option.
So a vote for BUILD would open up ALL possibilities afterward. The Tokens will be created and are then under the ownership and control of a System designed and governed by the community (yet we have to decide how this system should work before any tokens will be released into it by the node operators).
After those Tokens are in the Hand of the community, a majority decision that says "do nothing and wait for all the reclaimers to come up and claim their tokens" will be a possibility. If a vote decides to do this, it will be done. 
First of all, we have to decide to make those Tokens available in this network again, afterwards, it's on the community for what cases to use them.
About a minimum participation in the first vote. I am definitely on your side. Any vote that is not turning out to have at least 30% participation is not to be taken seriously. I would even hope to agree on 50% participation, but we will have to discuss this. Therefore voting needs to be done super easy and it needs to be promoted into the last part of the world that a unique possibility has come up. 
I think a vote "Do nothing? Ves? No?" will not be something a majority in the community will be interested in. It's on us to show which other amazing things can be achieved with these funds to create a demand and the wish to participate in this community governance process. Then we will see huge participation in the vote.

**Reply by AXBBXA** [reply]
> I think a vote "Do nothing? Ves? No?" will not be something a majority in the community will be interested in. It's on us to show which other amazing things can be achieved with these funds to create a demand and the wish to participate in this community governance process. Then we will see huge participation in the vote.

Thank you, Phylo. For the sake of good order, I did not suggest "Do nothing - Yes or No" (but "Should the Community assume control of the tokens - Yes or No") as a first vote. To me, this alternative ("Should the Community assume control of the tokens - Yes or No") appears to be both a more logical and easier to understand vote for the first question.

In contrast, the consequences of the "Build" vs "Burn" question may not be as easily understandable for people not following the discussion and it may give the impression to some that they are asked to choose between a rock and a hard place. It may result in less people participating in the vote than anticipated.

While I share the view that the opportunity to build a DAO is great and unique, questions should be phrased in a neutral way. They should not be biased towards a certain outcome. Also, questions should not combine different alternatives without being exhaustive and they should avoid pitting alternatives against each other that are not necessarily mutually exclusive.

Accordingly, my proposal for the first two votes would be:
1. Should the Community assume control of the tokens - Yes or No?
2. If yes: Should the Community build a DAO? Yes or No?

My impression is that the core community appears to have already made up its mind how the first question should be phrased. If this is not the case, I would kindly ask the core community to take my above comments into consideration.
